@Voskrese: ypur proposed changes fail in post build validation. The question is, if it is a real problem stat the static build is not supported under windows (even the error message in CI reads: "##[error] NEW ADDED PORT FAILURE: libcerf:x64-windows-static (ignore if triplet is not supported by upstream port)").

--- a/ports/libcerf/001-fix-static-build.patch
+++ b/ports/libcerf/001-fix-static-build.patch
@@ -1,0 +1,13 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index f79581f..4fc9457 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -56,6 +56,8 @@ if(MSVC)
+     # set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${PROJECT_BINARY_DIR}/bin/$<CONFIG>)
+     if(BUILD_SHARED_LIBS)
+         set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)
++    else()
++        set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} /MTd")
+     endif()
+ else()
+     add_compile_options(-O2 -Wno-sign-compare -fno-omit-frame-pointer)

--- a/ports/libcerf/CONTROL
+++ b/ports/libcerf/CONTROL
@@ -1,3 +1,4 @@
 Source: libcerf
 Version: 1.13
 Description: A self-contained numeric library that provides an efficient and accurate implementation of complex error functions, along with Dawson, Faddeeva, and Voigt functions.
+Homepage: https://jugit.fz-juelich.de/mlz/libcerf

--- a/ports/libcerf/portfile.cmake
+++ b/ports/libcerf/portfile.cmake
@@ -1,6 +1,10 @@
 include(vcpkg_common_functions)
 
-vcpkg_check_linkage(ONLY_DYNAMIC_LIBRARY ONLY_DYNAMIC_CRT)
+if(VCPKG_LIBRARY_LINKAGE STREQUAL dynamic)
+  set(BUILD_SHARED_LIBS ON)
+else()
+  set(BUILD_SHARED_LIBS OFF)
+endif()
 
 vcpkg_download_distfile(ARCHIVE
     URLS "https://jugit.fz-juelich.de/mlz/libcerf/uploads/924b8d245ad3461107ec630734dfc781/libcerf-1.13.tgz"
@@ -10,7 +14,8 @@ vcpkg_download_distfile(ARCHIVE
 
 vcpkg_extract_source_archive_ex(
     OUT_SOURCE_PATH SOURCE_PATH
-    ARCHIVE ${ARCHIVE} 
+    ARCHIVE ${ARCHIVE}
+    PATCHES 001-fix-static-build.patch
 )
 
 vcpkg_configure_cmake(
@@ -19,6 +24,7 @@ vcpkg_configure_cmake(
     OPTIONS
         -DCERF_CPP=ON
         -DLIB_MAN=OFF
+        -DBUILD_SHARED_LIBS=${BUILD_SHARED_LIBS}
 )
 
 vcpkg_install_cmake()

